src/styles/editedLabelStyles/types.ts

```diff
@@ -0,0 +1,5 @@
+import {CSSProperties} from 'react';
+
+type EditedLabelStyles = Partial<Pick<CSSProperties, 'alignItems' | 'display'>>;
```

Comment:
```suggestion
type EditedLabelStyles = TextStyle | CSSProperties;
```

We can relax a little bit in this one as well!

Reply:
Updated